app/src/utils/utils.ts

```diff
@@ -17,9 +17,11 @@ import {
   SUPPORTED_TOKENS_MAPPING,
   WETH_ADDRESS,
 } from 'src/utils/chains';
-import { getMetaPtr } from 'src/utils/data/ipfs';
 import { Ref } from 'vue';
 
+// --- Constants ---
+const retrievalEndpoint = 'https://scopelift.b-cdn.net/ipfs';
```

Comment:
Could we move this to `constants.ts` and import from there?

Reply:
We most certainly could